src/authentication/oauth2Auth.ts

```diff
@@ -300,6 +300,10 @@ export class Oauth2Auth extends AlfrescoApiClient {
     redirectLogin(): void {
         if (this.config.oauth2.implicitFlow && typeof window !== 'undefined') {
             let href = this.composeImplicitLoginUrl();
+            const afterLoginUriSegment = window.location.hash.split('&')[0].indexOf('session_state') === -1 ? window.location.hash.split('&')[0] : '';
```

Comment:
I think needs some safety checks. In case `&` does not exist, it will crash on getting `[0]`